lib/kb_Msuite/Utils/DataStagingUtils.py

```diff
@@ -55,22 +55,47 @@ def stage_input(self, input_ref, fasta_file_extension):
         #     the object.
         obj_name = input_info[1]
         type_name = input_info[2].split('-')[0]
+
+        # Standard Single Assembly
         if type_name in ['KBaseGenomeAnnotations.Assembly', 'KBaseGenomes.ContigSet']:
             au = AssemblyUtil(self.callback_url)
             os.makedirs(input_dir)
             filename = os.path.join(input_dir, obj_name + '.' + fasta_file_extension)
             au.get_assembly_as_fasta({'ref': input_ref, 'filename': filename})
             if not os.path.isfile(filename):
                 raise ValueError('Error generating fasta file from an Assembly or ContigSet with AssemblyUtil')
+            # make sure fasta file isn't empty
+            if self.count_fasta_seq_len(filename) == 0:
+                raise ValueError('Assembly or ContigSet is empty in filename: '+str(filename))
             pass
+
+        # AssemblySet
+        elif type_name == 'KBaseSets.AssemblySet':
+            raise ValueError('Cannot yet stage fasta file input directory from KBaseSets.AssemblySet')
+
+        # Binned Contigs
         elif type_name == 'KBaseMetagenomes.BinnedContigs':
             # download the bins as fasta and set the input folder name
             au = MetagenomeUtils(self.callback_url)
             bin_file_dir = au.binned_contigs_to_file({'input_ref': input_ref, 'save_to_shock': 0})['bin_file_directory']
             os.rename(bin_file_dir, input_dir)
+            # make sure fasta file isn't empty
             self.set_fasta_file_extensions(input_dir, fasta_file_extension)
+            for (dirpath, dirnames, filenames) in os.walk(input_dir):
```

Comment:
I don't see any test for this logic as well

Reply:
again, trivial, but ok I'll make a single unit test that exercises this stuff.

Reply:
added unit test that exercises this